test/gtest/uct/uct_test.cc

```diff
@@ -385,6 +385,8 @@ void uct_test::set_cm_resources(std::vector<resource>& all_resources)
 std::vector<const resource*> uct_test::enum_resources(const std::string& tl_name)
 {
     static bool tcp_fastest_dev = (getenv("GTEST_UCT_TCP_FASTEST_DEV") != NULL);
+    static const char *str      = getenv("GTEST_UCT_RSC_NAME");
```

Comment:
can't we rely on `UCX_NET_DEVICES`?

Reply:
i would like not to reuse existing options for other purpose and since I need to add the MD one, it makes sense to add the RSC one with same naming. Here we use/add `GTEST_UCT_*` to act in test code only.

Reply:
ok